src/qt/askpassphrasedialog.cpp

```diff
@@ -240,10 +234,6 @@ bool AskPassphraseDialog::eventFilter(QObject *object, QEvent *event)
                 ui->capsLabel->clear();
             }
         }
-    } else if (event->type() == QEvent::FocusIn) {
-        if (object->inherits("QLineEdit")) {
-            keyboard->setInput(object);
-        }
     }
     return false;
-}
+}
```

Comment:
While this is a bit of a legacy concern (most if not all compilers have been fixed to work anyway); used to be a new-line was required at the end of all files or else some compilers would crash.

As I said it's a legacy concern but it's also standard at this point to include a new line regardless.